PepperDashEssentials/Room/Types/EssentialsHuddleVtc1Room.cs

```diff
@@ -672,6 +692,35 @@ public static void AllRoomsOff()
 				(room as EssentialsHuddleSpaceRoom).RunRouteAction("roomOff");
 		}
 
+
+		/// <summary>
+		/// Setup the external sources for the Cisco Touch 10 devices that support IHasExternalSourceSwitch
+		/// </summary>
+		private void SetCodecExternalSources()
+		{
+			

```

Comment:
Instead of continuing the pattern of `(VideoCodec as IHasExternalSourceSwitching).someMethod`, let's create a local variable, cast to `IHasExternalSourceSwitching`, and check for null. If the local variable is null, return out of this method.